devdoc/thandle_requirements.md

```diff
@@ -163,4 +163,24 @@ Given a previously constructed `THANDLE(T)`, `THANDLE_GET_T(T)` reeturns a point
 
 **SRS_THANDLE_02_023: [** If `t` is `NULL` then `THANDLE_GET_T(T)` shall return `NULL`. **]**
 
-**SRS_THANDLE_02_024: [** `THANDLE_GET_T(T)` shall return the same pointer as `THANDLE_MALLOC`/`THANDLE_MALLOC_WITH_EXTRA_SIZE` returned at the handle creation time. **]**
\ No newline at end of file
+**SRS_THANDLE_02_024: [** `THANDLE_GET_T(T)` shall return the same pointer as `THANDLE_MALLOC`/`THANDLE_MALLOC_WITH_EXTRA_SIZE` returned at the handle creation time. **]**
+
+###     THANDLE_COPY_MACRO(T)
```

Comment:
>THANDLE_COPY [](start = 8, length = 12)

poetry: I actually find copy a bit misleading, because copy would make me think of copying the contents of an existing handle.
I would name this something like: create_from_content...
/poetry.

Reply:
Renames are cheap.

---
In reply to: [410621415](https://github.com/Azure/azure-c-util/pull/27#discussion_r410621415) [](ancestors = 410621415)

Reply:
poetrizied.